Formula/hashcat.rb

```diff
@@ -23,6 +23,12 @@ class Hashcat < Formula
 
   depends_on "gnu-sed" => :build
 
+  on_linux do
+    depends_on "opencl-headers" => :build
```

Comment:
This isn't used by default (Makefile uses `USE_SYSTEM_OPENCL=0`) and will use bundled copy